app/controllers/repos_controller.rb

```diff
@@ -85,10 +84,9 @@ def params_blank?
       repo_params.blank?
     end
 
-    def cached_repos(type, options = {})
-      Rails.cache.fetch("user/#{type}/#{current_user.id}", expires_in: 30.minutes) do
-        repos_response = GitHubBub.get("/user/#{type}", { token: current_user.token }.merge(options))
-        SortedRepoCollection.new(repos_response.json_body)
+    def cached_repos(kind, options = {})
+      Rails.cache.fetch("user/#{kind}/#{current_user.id}", expires_in: 30.minutes) do
+        SortedRepoCollection.new(GithubFetcher::Repo.repos_for(current_user.token, kind, options))
```

Comment:
Same thing here. Basically I think GithubFetcher::Repo should only be accessed directly by a model, which is composing it.